<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-809396K" failures="1" tests="1" name="coyni_mobile.tests.SignUpTest.testCreateAccountInvalidData" time="27.551" errors="0" timestamp="2021-12-15T00:35:21 IST">
  <testcase classname="coyni_mobile.tests.SignUpTest" name="testCreateAccountInvalidData" time="26.621">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at ilabs.mobile.reporting.ExtentTestManager.setFailMessageInReport(ExtentTestManager.java:72)
at coyni_mobile.utilities.CommonFunctions.validateFormErrorMessage(CommonFunctions.java:50)
at coyni_mobile.tests.SignUpTest.testCreateAccountInvalidData(SignUpTest.java:105)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at ilabs.MobileFramework.TestReader.run(TestReader.java:55)
at ilabs.MobileFramework.Runner.runOneIteration(Runner.java:149)
at ilabs.MobileFramework.Runner.runRangeOfIterations(Runner.java:113)
at ilabs.MobileFramework.Runner.run(Runner.java:64)
at coyni_mobile.runner.TestRunner.main(TestRunner.java:9)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testCreateAccountInvalidData -->
  <testcase classname="coyni_mobile.tests.SignUpTest" name="testFieldValidationCreateAccount" time="0.0">
    <ignored/>
  </testcase> <!-- testFieldValidationCreateAccount -->
  <testcase classname="coyni_mobile.tests.SignUpTest" name="testSignUp" time="0.0">
    <ignored/>
  </testcase> <!-- testSignUp -->
  <testcase classname="coyni_mobile.tests.SignUpTest" name="testSignUpWithInvalidData" time="0.0">
    <ignored/>
  </testcase> <!-- testSignUpWithInvalidData -->
</testsuite> <!-- coyni_mobile.tests.SignUpTest.testCreateAccountInvalidData -->
